src/viadot/orchestration/prefect/tasks/bigquery.py

```diff
@@ -0,0 +1,72 @@
+"""'bigquery.py'."""
+
+import contextlib
+
+import pandas as pd
+from prefect import task
+
+from viadot.orchestration.prefect.exceptions import MissingSourceCredentialsError
+from viadot.orchestration.prefect.utils import get_credentials
+
+
+with contextlib.suppress(ImportError):
+    from viadot.sources import BigQuery
```

Comment:
This shouldn't be here as the source doesn't use any optional dependencies.

Reply:
[6d00a97](https://github.com/dyvenia/viadot/pull/1035/commits/6d00a97a1f690365491fb1886e6b2fabcce79f2d)